package.json

```diff
@@ -3,8 +3,8 @@
   "version": "1.0.0",
   "private": "private",
   "scripts": {
-    "start": "node ./bin/www",
-    "test": "NODE_ENV=test npm test"
+    "start": "npm start",
```

Comment:
What's the reason for this change? It was fine before, but now you are referencing `npm start` from `npm start`, creating an endless loop